.gitignore

```diff
@@ -16,6 +16,7 @@ images/person_image7.jpg
 images/person_image8.jpg
 images/person_image12.jpg
 images/person_image13.jpg
+examples/singlemodel/annotated_video.mp4
```

Comment:
I have a feeling that this kind of list of files is hard to maintain. May be we can modify samples so they will store all outputs in some dedicated dir? In this case we can put just this dir into .gitignore
I did it once, when I created workarea dir for that purpose (and put it in .gitignore)

Reply:
@vlad-nn agree with your comment. However, I don't know how a directory or a directory structure 
will work in colab.